pkgs/tools/graphics/sanjuuni/tests/run-on-nixos-artwork.nix

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
`runCommand` is more appropriate here.

Reply:
Definitely, I've modified the test to use `runCommand`.